flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_ATC_ClearanceReq.js

```diff
@@ -0,0 +1,107 @@
+class CDUAtcClearanceReq {
+    static CreateDataBlock() {
+        return {
+            clearance: false
+        };
+    }
+
+    static CanSendData(data) {
+        return data.clearance;
+    }
+
+    static CreateRequest(mcdu) {
+        const retval = new Atsu.CpdlcMessage();
+        retval.Station = mcdu.atsu.atc.currentStation();
+        retval.Content.push(Atsu.CpdlcMessagesDownlink["DM25"][1].deepCopy());
+        retval.Content[0].Content[0].Value = "DEPARTURE";
+        return retval;
+    }
+
+    static ShowPage(mcdu, title, data = CDUAtcClearanceReq.CreateDataBlock()) {
+        mcdu.clearDisplay();
+        mcdu.page.Current = mcdu.page.ATCGroundRequest;
```

Comment:
This is probably not the best way to go about modifying a parameter object's attribute but it's what we have already been doing so we will review this as part of a MCDU overhaul than a specific change to this PR